src/main/resources/swagger.api/schemas/authority-storage/authorityDto.yaml

```diff
@@ -168,22 +148,22 @@ properties:
     type: array
     description: See also from tracing term that represents broader, more general concepts related to the authority record
     items:
-      type: string
+      $ref: './authorityRelatedHeading.yaml'
   saftNarrowerTerm:
     type: array
     description: See also from tracing term that that represents narrower, more specific concepts derived from the authority record
     items:
-      type: string
+      $ref: './authorityRelatedHeading.yaml'
   saftEarlierHeading:
     type: array
     description: See also from tracing heading that was previously used to represent the concept or entity described by the authority record. This field is used to track the evolution of terms or headings over time, facilitating the linking of historical and current data.
     items:
-      type: string
+      $ref: './authorityRelatedHeading.yaml'
   saftLaterHeading:
     type: array
     description: See also from tracing heading that replaced the current heading used in the authority record. This field helps in maintaining the continuity of catalog records by linking past headings to their more current versions.
     items:
-      type: string
+      $ref: './authorityRelatedHeading.yaml'
   identifiers:
```

Comment:
If it's just one saftBroaderTerm, saftNarrowerTerm, saftEarlierHeading, saftLaterHeading then should it be just broaderTerm, narrowerTerm, earlierHeading, laterHeading ?